packages/core/src/fhir-deduplication/__tests__/group-same-diag-reports.test.ts

```diff
@@ -12,31 +12,68 @@ let diagReportId: string;
 let diagReportId2: string;
 let diagReport: DiagnosticReport;
 let diagReport2: DiagnosticReport;
+let practId: string;
+let practId2: string;
+let practRef: Reference<Practitioner>;
+let practRef2: Reference<Practitioner>;
 
 beforeEach(() => {
+  practId = faker.string.uuid();
+  practId2 = faker.string.uuid();
+
+  practRef = { reference: `Practitioner/${practId}` };
+  practRef2 = { reference: `Practitioner/${practId2}` };
+
   diagReportId = faker.string.uuid();
   diagReportId2 = faker.string.uuid();
-  diagReport = makeDiagnosticReport({ id: diagReportId });
-  diagReport2 = makeDiagnosticReport({ id: diagReportId2 });
+  diagReport = makeDiagnosticReport({ id: diagReportId, result: resultExample });
+  diagReport2 = makeDiagnosticReport({ id: diagReportId2, presentedForm: presentedFormExample });
```

Comment:
Same comment from another PR: I'd rather have each test calling a function that builds these things, so its clear there where they come from, than having a sort of shared state across all tests that we might or not be updating.

Not a blocker for this PR, as we have this in multiple places, but I think we should avoid this if we can - unless it's to `spyOn` and build mocked stuff.